docs/modules/ROOT/pages/backup-and-restore.adoc

```diff
@@ -33,8 +33,8 @@ When you have at least one backup, you can use it to bootstrap a new cluster.
 
 . Click *Restore* in the row of your selected backup.
 
-. When the details of your new cluster are displayed, you can update its deployment region or cluster name.
+. When the details of your new cluster are displayed, you can update its cluster name.
 
-. Click *Restore Data to a New Cluster* to deploy your new cluster.
+. Click *Restore Data to a New Cluster* to deploy your new cluster. To check the Hazelcast version of the new cluster, go to the dashboard and view the *Cluster Details*.
```

Comment:
> To check the Hazelcast version of the new cluster, go to the dashboard and view the *Cluster Details*.

This is not a guarantee a restored cluster will be of the same version. If we released a new default image since the backup was taken, it will run the newer version. I suggest we leave the old wording to avoid confusion.